controllers/secret_controller.go

```diff
@@ -382,6 +382,15 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 		{Receiver: receiverMakeItWarning, Match: map[string]string{"severity": "critical", "namespace": "openshift-deployment-validation-operator"}},
 	}
 
+	// Silence insights in FedRAMP until its made available in the environment
+	// https://issues.redhat.com/browse/OSD-13685
+	if config.IsFedramp() {
+		pagerdutySubroutes = append(pagerdutySubroutes,
+			[]*alertmanager.Route{
+				{Receiver: receiverNull, Match: map[string]string{"alertname": "ClusterOperatorDown", "name": "insights"}},
+			}...)
```

Comment:
Since there's only one exception I think we should just do:
```go
	if config.IsFedramp() {
		pagerdutySubroutes = append(pagerdutySubroutes,
			&alertmanager.Route{Receiver: receiverNull, Match: map[string]string{"alertname": "ClusterOperatorDown", "name": "insights"}},
		)
	}
```